packages/api/tests/routes.test.js

```diff
@@ -0,0 +1,13 @@
+const request = require('supertest')
+const app = require('../server')
+describe('Post Endpoints', () => {
+  it('should create a new post', async () => {
```

Comment:
I know this method is just temporary, but the descriptions here seem a little.. confusing?

Reply:
Changed Post to Get to reflect the test better